source/scss/_pf-basic-icons.scss

```diff
@@ -219,3 +219,12 @@
 .#{$pf-icon-prefix}-zone:before {
   content: $pficon-var-zone;
 }
+.pf-icon-circle-lg {
```

Comment:
For our class names we'll use BEM, following the structure: `{{pf-block}}__{{element}}--{{modifier}}`
The size is a modifier, but is `circle` and element?
@jgiardino @srambach: what do you think? ?